### PR TITLE
Integrate daemon

### PR DESCRIPTION
**Pull request by jkarni**
This switches all of our machines to IPFS.

It also renames a lot of "chain" to "machine", and removes `:enter-chain`.

### CODE REVIEW

**Comment by jkarni** [comment]
Got a few lingering chain.rad to fix.

File: src/Radicle/Internal.hs [hunk 1]
```diff
@@ -13,3 +16,8 @@ import           Radicle.Internal.Parse as X
 import           Radicle.Internal.Pretty as X
 import           Radicle.Internal.PrimFns as X
 import           Radicle.Internal.Type as X
+
+createBindings :: (MonadIO m, ReplM m) => [Text] -> IO (Bindings (PrimFns m))
```
Comment (geigerzaehler) [inline src/Radicle/Internal.hs]: Maybe we should call this `createReplBindings` or `createImpureBindings`. Please also add documentation.

File: exe/ReferenceDoc.hs [hunk 2]
```diff
@@ -33,13 +34,15 @@ instance FromJSON Content
 main :: IO ()
 main = do
     content <- decodeFileThrow "reference-doc.yaml"
+    daemonClientPrimFns <- createDaemonClientPrimFns
+    let bindings = addPrimFns (replPrimFns [] <> daemonClientPrimFns) pureEnv
```
Comment (geigerzaehler) [inline exe/ReferenceDoc.hs]: Can’t we use `createBindings` from `Radicle.Internal`?

File: exe/Doc.hs [hunk 3]
```diff
@@ -26,7 +27,9 @@ run f = do
     let extensions = def { readerExtensions = githubMarkdownExtensions }
     pand <- runIOorExplode $ readMarkdown extensions txt
     let code = T.append "(load! (find-module-file! \"prelude.rad\"))" (getCode pand)
-    res <- runLang (replBindings []) $ interpretMany (toS f) $ code
+    daemonClientPrimFns <- createDaemonClientPrimFns
+    let bindings = addPrimFns (replPrimFns [] <> daemonClientPrimFns) pureEnv
```
Comment (geigerzaehler) [inline exe/Doc.hs]: Can’t we use `createBindings` from `Radicle.Internal`?

**Pull request merged**